utilities/infra.py

```diff
@@ -102,6 +103,27 @@ def create_ns(
             yield ns
 
 
+def wait_for_replicas_in_deployment(deployment: Deployment, replicas: int) -> None:
+    """
+    Wait for replicas in deployment to updated in spec.
+
+    Args:
+        deployment (Deployment): Deployment object
+        replicas (int): number of replicas to be set in spec.replicas
+
+    Raises:
+        TimeoutExpiredError: If replicas are not updated in spec.
+
+    """
+    for sample in TimeoutSampler(
```

Comment:
This might be a good candidate for `@retry`. Also do you want to poll every seconds? That seems too many api calls.

Reply:
>Also do you want to poll every seconds? That seems too many api calls.
yes. do not wait to wait here./